apps/blockchain/lib/blockchain/configuration/homestead_test.ex

```diff
@@ -0,0 +1,15 @@
+defmodule Blockchain.Configuration.HomesteadTest do
+  defstruct contract_creation_cost: 53_000, has_static_call: true
```

Comment:
Does homestead have the static call opcode (I thought it was introduced later) or does this refer to something else?

Reply:
Yes, my bad. `delegatecall` was introduced in Homestead